assembl/static2/js/app/components/administration/discussion/manageProfileOptionsForm.jsx

```diff
@@ -118,9 +118,18 @@ const ManageProfileOptionsForm = ({
   return (
     <div className="admin-box">
       <SectionTitle title={I18n.t('administration.discussion.3')} annotation={I18n.t('administration.annotation')} />
+      <div className="intro-text">
+        <Translate value="administration.profileOptions.toggleLegalContentIntro" />
+      </div>
+      <div className="legal-content-section form-container">
+        <Checkbox className="textfield-checkbox">
+          <Translate value="administration.profileOptions.LegalContentButton" />
```

Comment:
Not very important but I prefer `toggleLegalContentsIntro` and `LegalContentsButton`